src/vivarium_census_prl_synth_pop/components/immigration.py

```diff
@@ -29,10 +29,7 @@ def setup(self, builder: Builder):
         immigrants = persons_data[persons_data["immigrated_in_last_year"]]
 
         is_gq = immigrants["relation_to_household_head"].isin(
-            [
-                "Insitutionalized GQ pop",
-                "Noninstitutionalized GQ pop",
-            ]
+            data_values.GROUP_QUARTER_IDS.keys()
```

Comment:
I thought this was supposed to key off of the new housing type column?

Reply:
That column is in the households, not persons, data. In other places, we use the relationship column for this purpose when working with the persons data. I suppose to be more technically correct we could always join to the households data and use the newly-loaded column, but that might overcomplicate things.

Just to note: the reason for adding that column in #167 was not to use it for immigration; it was to fix an unrelated bug in our handling of the households data.

Reply:
To clarify, this column should be correct once we're in the simulation. The issue is that we couldn't rely on this column in the source ACS data-set. Am I understanding this correctly? If so, feel free to merge.

Reply:
Nope not quite, and I see how this is confusing.

I think what you said is a clearer way to think about it, and I will update this code to use the newly-loaded housing type column from ACS.